### ISSUE TITLE
include `text` in sentences.data()

### ISSUE THREAD

**Issue by giorgio79** [spencermountain/compromise#483]
Hello,

I tried to extract the sentences from a text, but compromise provides a normal form only. Can we get the original text as well? I am only looking for a simple sentence array.

IMHO lowercasing stuff whichever way is definitely a bad idea :)
```
var compromise = require('compromise') 
var doc = compromise('London is calling. I love you.')
var sentences = doc.sentences().out('array')
console.log(sentences);

nlp('hi everybody! Hi Dr. Nick!').sentences().data().length
//[{normal:'hi everybody'}, {normal:'hi dr nick'}]
```

Here is what I am looking for https://www.npmjs.com/package/sentence-extractor , hoped compromise can do that :)

**Comment by spencermountain** [comment]
hey György, yeah i get what you're saying. the plaintext version should definitely be in the sentences.data() response.
i'll leave this open until I add that.

you can do this, for now:
https://runkit.com/spencermountain/5b043b2941a474001204884f
cheers

**Comment by spencermountain** [comment]
hey György, fixed in 11.9.1
